src/actions/index.js

```diff
@@ -142,6 +143,34 @@ export function fetchProjectsByParms(tag) {
   }
 }
 
+export function fetchWorkflowsForMobile(projects) {
+  return (dispatch) => {
+    return new Promise((resolve) => {
+      const swipeProjects = filter((proj) => { return find(propEq('projectID', proj.id), SWIPE_WORKFLOWS) }, projects)
+      const getWorkflows = (project) => {dispatch(fetchProjectWorkflows(project))}
+      forEach(getWorkflows, swipeProjects)
+      return resolve()
+    })
+  }
+}
+
+export function fetchProjectWorkflows(project) {
+  return dispatch => {
+    return new Promise((resolve) => {
+      apiClient.type('projects').get({id: project.id}).then((projects) => {
```

Comment:
I believe you can also put 'projects' in brackets to avoid the next line using `head(projects)`. 

`.then(([projects]))` -> would give you only item in that array. Just for future reference.